### PR TITLE
Milestone 4

### PR DESCRIPTION
**Pull request by IBTechRaj**
Finished requirements of Milestone-4.
-This milestone is created from Milestone-3. I think, after approval, I have to merge with Milestone-3. Please advise.

Thanks

I was getting error 'match' not found when setting up Eslint and StyleLint with 'npm'.  I used 'yarn' and it worked. There were no errors when I ran 'npx eslint' also. But here in Github I am finding errors.

Kindly advise

### CODE REVIEW

**Review by Redvanisation — CHANGES_REQUESTED** [review]
# Changes required  ♻️ 

### Hello team  🌟

**Nice work on this project, but the following changes need to be done before moving on ⬇️** 

- Nicely done however the linter should be passing, please fix that.

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Comment by IBTechRaj** [comment]
Hi, I have fixed all linting errors and style error in local repo and pushed it. But here in Github I am finding the errors again. 

I need help. I am not able to connect with my reviewer. That is why requesting another code review.  

Thanks

**Review by fatymahmed — CHANGES_REQUESTED** [review]
### Hi @IBTechRaj,
- [ ] Make sure that linter checks are passing.

- The following is marked as `optional` because it was not mentioned in the first review but I highly suggest you to make the change

[OPTIONAL]
- The project should have a descriptive README.md [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from Pathwright
### Status: Changes Required ♻️
_Feel free to reach out to me on slack (Fatima), if you have any questions_

**Comment by fatymahmed** [comment]
> Hi, I have fixed all linting errors and style error in local repo and pushed it. But here in Github I am finding the errors again.
> 
> I need help. I am not able to connect with my reviewer. That is why requesting another code review.
> 
> Thanks

Hi @IBTechRaj, I've reached out to you on slack regarding this.

**Comment by IBTechRaj** [comment]
I have done all the corrections suggested. Also thanks for the help from Fatima Ahmed. 
Submitting for re-review.

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  try {
+    switch (operator) {
+      case '+':
+        return a.plus(b).toString();
+      case '-':
+        return a.minus(b).toString();
+      case 'X':
+        return a.times(b).toString();
+      case '÷':
+        return a.div(b).toString();
```
Comment (shloch) [inline src/logic/operate.js:19]: dont forget to manage DIVISION by ZERO on `milestone5`, else it will generate an error

**Review by shloch — APPROVED** [review]
## Hello @IBTechRaj 

### You get a thumbs,  but please we'll have to reject any submission with git-branch conflicts.

Screen Shot 2020-06-19 at 1 58 15 PM

The base branch should be `development`, every other milestone-branch should branch out from `development`
Screen Shot 2020-06-19 at 1 56 05 PM

Dont forget to re-read if need be [GIT-FLOW guideline](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59882084/)

git

### Happy coding

**Comment by IBTechRaj** [comment]
Resolved conflicts in local repo, merged and pushed to github

**Pull request closed without merging**